pkg/install/resources.go

```diff
@@ -0,0 +1,53 @@
+package install
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	rbacv1beta1 "k8s.io/api/rbac/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func labels() map[string]string {
+	return map[string]string{
+		"component": "ark",
+	}
+}
+
+func objectMeta(namespace, name string) metav1.ObjectMeta {
+	return metav1.ObjectMeta{
+		Name:      name,
+		Namespace: namespace,
+		Labels:    labels(),
+	}
+}
+
+func ServiceAccount(namespace string) *corev1.ServiceAccount {
+	return &corev1.ServiceAccount{
+		ObjectMeta: objectMeta(namespace, "ark"),
+	}
+}
+
+func ClusterRoleBinding(namespace string) *rbacv1beta1.ClusterRoleBinding {
```

Comment:
this should be cluster-scoped, so no `namespace` arg

Reply:
Cluster Roles are cluster-scoped, but bindings are namespace scoped afaik.

Reply:
@marpaia is correct - it's the namespace for the ark service account. We could make the service account's name configurable too but I don't think we need that right now.

Reply:
Yes they are cluster-scoped but you need the namespace for the subject. We do _not_ need it for the object meta

Reply:
Got it, that's updated now.